### DISCUSSION TITLE
Trouble after deploy app new version with PWA

### DISCUSSION THREAD

**Discussion by matheus-foscarinid** [quasarframework/quasar#16811] [CLI - PWA mode] [answered]
Hey everyone,

I'm currently facing an issue with my Progressive Web App (PWA) whenever I deploy a new update to my web application. The problem arises during the first load after the update, where I encounter a 404 error for my build files.

![Screenshot](https://github.com/quasarframework/quasar/assets/57161520/fe3423b3-0293-4429-822d-cd0c5218cb12)

Upon reloading the app, it successfully reflects the updated version. I suspect that the service worker might take some time to update, but the main concern is why it attempts to access the old versions on the initial load, which should ideally be cached.

I've spent considerable time investigating this issue, but I'm struggling to understand why it isn't retaining the old version.

Some additional information that might be useful:
- I deploy my system using Docker.
- The system is not hosted on the root URL; it resides in the /app/ path.

My quasar.conf file:

```
    ssr: {
      pwa: true,
    },

    // https://quasar.dev/quasar-cli/developing-pwa/configuring-pwa
    pwa: {
      workboxPluginMode: 'GenerateSW', // 'GenerateSW' or 'InjectManifest'
      workboxOptions: {
        skipWaiting: true,
        clientsClaim: true
     }
    },
```

Any insights or guidance would be greatly appreciated!

Thanks in advance!

**Comment by anatoliynovo** [comment]
@matheus-foscarinid did you find out what caused it?

**Reply by matheus-foscarinid** [reply, marked as answer]
My bad, I forgot to add an update here. Yes, took us some time to figure out but basically our css minifying was passing the caching limit for PWA. We fixed the issue with this config `maximumFileSizeToCacheInBytes`, we have an open issue to take a look why the generated css file was so big.
